Core/clim-core/standard-presentations.lisp

```diff
@@ -149,6 +165,33 @@
   nil)
 
 
+;;; *NULL-PRESENTATION* is provided as something what the user may
+;;; want to return (or pass somewhere else), clicking on the area
+;;; without a presentation in a context involving blank-area will
+;;; create a distinct presentation instance with more information, in
+;;; particular the event sheet and pointer position of the click.
+;;;
+;;; The object may not be fully initialized as a presentation. Do
+;;; other slots have to be bound in order for it to be useful?  Guess
+;;; we'll see.
+
+(defvar *null-presentation*
+  (make-instance 'standard-presentation :object nil
+                                        :type 'blank-area
+                                        :view +textual-view+))
+
+(defun make-blank-area-presentation (sheet x y event)
+  (if (null sheet)
+      *null-presentation*
+      (let ((event (or event
+                       (make-instance 'pointer-event :sheet sheet :x x :y y))))
+        (multiple-value-bind (x1 y1 x2 y2) (bounding-rectangle* sheet)
+          (make-instance 'standard-presentation
+                         :object event
+                         :type `((blank-area :sheet ,sheet :region ,(make-point x y))
+                                 :description "Blank Area")
+                         :view +textual-view+
+                         :x1 x1 :y1 y1 :x2 x2 :y2 y2)))))
```

Comment:
this function could accept only the event (and the caller would be responsible for synthesizing it)

Reply:
also this would be easier to read as:

```
(when (null event)
  (return-from … *null-presentation*))
(let ((sheet (event-sheet …) …)
  (multiple-value-bind …))
```